internal/discogs.go

```diff
@@ -28,13 +28,17 @@ func getToken() string {
 
 	token, err := os.ReadFile(ConfigDir + "token")
 	ErrorCheck(err)
+	token = bytes.ReplaceAll(token, []byte("\n"), []byte(""))
 
 	return string(token)
 }
 
 // BarcodeSearch searches the discogs database (using the provided barcode)
 // and returns the release struct
 func BarcodeSearch(barcode string, interactive bool) *discogs.Release {
+	if client == nil {
```

Comment:
Might be nice to just do this in the init? I dunno if this is something you want to do. Either way its fine.